app/models/salesforce/case.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ::Salesforce
+
+  class Case < ::ActiveRecord::Base
+    self.table_name = "salesforce_cases"
+
+    belongs_to :topic
+
+    def create!
```

Comment:
Generally, I think we should avoid overriding ActiveRecord methods since it may be confusing for someone else reading through the code in the future.

Reply:
I'm not overriding. It's an instance method. Anyway, I changed the name to prevent confusions :+1: